website/i18n/en.json

```
@@ -408,17 +417,21 @@
         "title": "Multi-value dimensions"
       },
       "querying/multitenancy": {
-        "title": "Multitenancy considerations"
+        "title": "Multitenancy considerations",
+        "sidebar_label": "Multitenancy"
       },
       "querying/post-aggregations": {
-        "title": "Post-Aggregations"
+        "title": "Postaggregations"
```

is this intentional? In the page text, it's mostly hyphenated, but sometimes two words. Or we could use the code form like groupBy, etc, which would then be "postAggregation".. okay, but not preferred, I'd say 

```suggestion
        "title": "postAggregation"
```

---

What do you think about "Post-aggregations"?

This text will appear in the sidebar, where starting with a lowercase would look weird.

---

"Post-aggregations" works for me.